lighthouse-core/scripts/legacy-javascript/summary-sizes.txt

```diff
@@ -0,0 +1,249 @@
+   7941 variants/core-js-3-only-polyfill/es6-reflect-apply/main.bundle.min.js
+   7906 variants/core-js-3-only-polyfill/es6-math-tanh/main.bundle.min.js
+   7905 variants/core-js-3-only-polyfill/es6-math-acosh/main.bundle.min.js
+   7875 variants/core-js-3-only-polyfill/es6-number-is-safe-integer/main.bundle.min.js
+   7820 variants/core-js-3-only-polyfill/es6-math-expm1/main.bundle.min.js
+   7814 variants/core-js-3-only-polyfill/es6-array-is-array/main.bundle.min.js
+   7785 variants/core-js-3-only-polyfill/es6-math-hypot/main.bundle.min.js
+   7762 variants/core-js-3-only-polyfill/es6-reflect-prevent-extensions/main.bundle.min.js
+   7751 variants/core-js-3-only-polyfill/es6-math-imul/main.bundle.min.js
+   7742 variants/core-js-3-only-polyfill/es6-number-is-integer/main.bundle.min.js
+   7701 variants/core-js-3-only-polyfill/es6-math-cbrt/main.bundle.min.js
+   7692 variants/core-js-3-only-polyfill/es6-reflect-is-extensible/main.bundle.min.js
+   7653 variants/core-js-3-only-polyfill/es6-math-log1p/main.bundle.min.js
+   7630 variants/core-js-3-only-polyfill/es6-math-sign/main.bundle.min.js
+   7627 variants/core-js-3-only-polyfill/es6-math-asinh/main.bundle.min.js
+   7607 variants/core-js-3-only-polyfill/es6-object-define-property/main.bundle.min.js
+   7591 variants/core-js-3-only-polyfill/es6-math-atanh/main.bundle.min.js
+   7550 variants/core-js-3-only-polyfill/es6-math-clz32/main.bundle.min.js
+   7538 variants/core-js-3-only-polyfill/es6-reflect-has/main.bundle.min.js
+   7532 variants/core-js-3-only-polyfill/es6-math-cosh/main.bundle.min.js
+   7530 variants/core-js-3-only-polyfill/es6-math-trunc/main.bundle.min.js
+   7529 variants/core-js-3-only-polyfill/es6-number-min-safe-integer/main.bundle.min.js
+   7528 variants/core-js-3-only-polyfill/es6-number-max-safe-integer/main.bundle.min.js
+   7522 variants/core-js-3-only-polyfill/es6-math-log10/main.bundle.min.js
+   7515 variants/core-js-3-only-polyfill/es6-math-log2/main.bundle.min.js
+   7505 variants/core-js-3-only-polyfill/es6-date-now/main.bundle.min.js
+   7500 variants/core-js-3-only-polyfill/es6-number-epsilon/main.bundle.min.js
+   6917 variants/core-js-3-only-polyfill/es6-object-to-string/main.bundle.min.js
+   5993 variants/core-js-3-only-polyfill/es6-function-has-instance/main.bundle.min.js
+   5992 variants/core-js-3-only-polyfill/es6-date-to-string/main.bundle.min.js
+   5421 variants/core-js-3-only-polyfill/es6-date-to-primitive/main.bundle.min.js
+   4773 variants/core-js-3-only-polyfill/es6-array-species/main.bundle.min.js
+   3410 variants/core-js-3-only-polyfill/es6-function-name/main.bundle.min.js
+
+
+variants/core-js-3-preset-env-esmodules
+3567 total
+2259 variants/core-js-3-preset-env-esmodules/false/main.bundle.min.js
+1308 variants/core-js-3-preset-env-esmodules/true/main.bundle.min.js
+
+
+variants/only-plugin
+2770 total
+1152 variants/only-plugin/-babel-plugin-transform-spread/main.bundle.min.js
```

Comment:
`only-plugin` sizes will be more interesting when I pull in a real app (prox?). will do for opportunity work.